profile/profiler.go

```diff
@@ -427,6 +430,10 @@ func (c *CPUInfo) GetSibling(processor int) (int, error) {
 	}
 
 	core := c.sockets[proc.socket].cores[proc.core]
+	if len(core.processors) == 1 {
```

Comment:
please add a comment explaining what this check means

Reply:
Done

Reply:
thanks, please push it for a review

Reply:
Done